web/js/tasks.js

```diff
@@ -1351,13 +1346,19 @@ Ext.onReady(function(){
 
                     // Create and populate a record
                     var newTask = new taskRecord();
-                    newTask.set('projectId', templateValues['projectId']);
                     if(templateValues && templateValues['ttype'] && !taskTypeStore.data.items.some(x => x.id == templateValues['ttype'])){
                       let message = `Task type of ${templateValues['ttype']} is not valid. The task type may have been deactivated. Please choose another task type.`
                       App.setAlert(false, message);
                       } else {
                         newTask.set('ttype', templateValues['ttype']);
                     }
+
+                    if(templateValues && !projectsList.some(x => x.id == templateValues['projectId'])){
+                        let message = `You are not assigned to this project. Please select another project or contact the system adminstrators.`
```

Comment:
Administrators is missing an 'i' here: 
`adminstrators` => `administrators`

Reply:
oops, thanks!